src/store/recommendation/sagas.ts

```diff
@@ -13,27 +13,27 @@ export function* getRecommendedTracks(
   genre: string,
   exclusionList: number[],
   currentUserId: Nullable<ID>
-): Generator<any, Track[], any> {
-  const tracks = yield apiClient.getRecommended({
+) {
+  const tracks = yield* call([apiClient, apiClient.getRecommended], {
     genre,
     exclusionList,
     currentUserId
   })
-  yield call(processAndCacheTracks, tracks)
-  return tracks as Track[]
+  yield* call(processAndCacheTracks, tracks)
+  return tracks
 }
 
-export function* getLuckyTracks(limit: number): Generator<any, Track[], any> {
-  const latestTrackID = yield call(Explore.getLatestTrackID)
+export function* getLuckyTracks(limit: number) {
+  const latestTrackID = yield* call(Explore.getLatestTrackID)
   const ids = Array.from({ length: limit }, () =>
     Math.floor(Math.random() * latestTrackID)
   )
-  const tracks = yield call(AudiusBackend.getAllTracks, {
+  const tracks: UserTrack[] = yield* call(AudiusBackend.getAllTracks, {
```

Comment:
also would be great to add a d.ts file for AudiusBackend, happy to help get that set up with you

Reply:
tbh not sure it's worth it as we have some stuff in the roadmap around libs improvements already, which should include types.